### ISSUE TITLE
Lua 5.3 not detected correctly

### ISSUE THREAD

**Issue by fsfod** [WheretIB/LuaDkmDebugger#4]
I was getting no callstacks in the debugger because the version detection logic didn't work for me for a Lua 5.3 dll. I managed to get it working by just checking if a [lua_rotate function is declared](https://github.com/fsfod/LuaDkmDebugger/commit/2597bcc095f16d9d16498dc7eb007f20a9bd680c).

**Comment by fsfod** [comment]
It looks like real fix is to always use the value from stateAddress instead of trying to use a optimized L in expressions.

```
long? version = EvaluationHelpers.TryEvaluateNumberExpression($"(int)*((lua_State*){stateAddress})->l_G->version", inspectionSession, thread, frame, DkmEvaluationFlags.TreatAsExpression | DkmEvaluationFlags.NoSideEffects);
```

**Comment by WheretIB** [comment]
Great idea, I've already had a fallback register lookup for state value, but didn't update the other two expressions.

**Comment by WheretIB** [comment]
Fixed in e611e404a864a7d5c04f600d869c9f8e43a99533

I will try to update the extension later on July 6th.
